src/honkalculate.template.html

```diff
@@ -9,6 +9,24 @@
     <link type="text/css" rel="stylesheet" href="https://cdn.datatables.net/colvis/1.1.1/css/dataTables.colVis.css" />
     <link type="text/css" rel="stylesheet" href="./js/vendor/select2/select2.css" />
     <link type="text/css" rel="stylesheet" href="./css/main.css?" />
+    <!-- Google Analytics -->
+    <script>
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+        ga('create', 'UA-XXXXXX-Y', 'calc.pokemonshowdown.com');
```

Comment:
@Zarel - please set up the property for this.

Reply:
We got clogged up on whether or not this should be a subdomain or subdirectory. I guess separate property is fine, though.

Reply:
You had [agreed to calc.pokemonshowdown.com](https://discordapp.com/channels/@me/551083939561865238/607642014280056839), and the rename issue (#332) assigned to you covers moving to a subdomain.

Reply:
Yes, that's why separate property is ifne.